# Dumper control messages.

# Velocity of the dumper, on a scale from -1 (lowering at full
# throttle) to 1 (raising at full throttle).
float64 vel
